### DISCUSSION TITLE
为deepin-terminal增加透明与不透明切换的快捷键功能

### DISCUSSION THREAD

**Discussion by liwl1991** [linuxdeepin/developer-center#3939] [Features Request & Ideas | 特性请求 & 头脑风暴]
开发组你们好：
在v20系列中，我为了能够高效地使用终端透明调整，自己下载了代码修改，在deepin-terminal设置选项中增加了一个快捷键设置，该快捷键能够快速切换终端透明和不透明状态，类似xshell里面的alt+r功能。该功能主要适用于终端窗口在前，其他窗口在后的场景。因为当前版本存在使用滑杆调节透明度的功能，我个人觉得这种调节比较慢速，想增加快捷键来操作。不清楚我这种修改的pr是否可以被合并？

**Comment by Decodetalkers** [comment]
@ArchieMeng cc

**Comment by Iatbzh** [comment]
cc,强烈支持，都v23了

**Reply by ArchieMeng** [reply]
啊这，你也需要吗？调透明度的接口就在那里。你可以自己改改试试。

**Reply by Iatbzh** [reply]
方便指引一下吗，文档哪里呢或者代码

**Reply by ArchieMeng** [reply]
> 方便指引一下吗，文档哪里呢或者代码

请参考这个提交 https://github.com/linuxdeepin/deepin-terminal/commit/7f1691bcc681c722748efc5e4da0e849230be132 。快捷键的应该是在另一个shortcutmanager文件里面。
